docs/guide/nested-fields.rst

```diff
@@ -283,6 +283,61 @@ populator of the relationship.
 If you don't have an existing type for creating a user, e.g. the "CreateCatInput" we used above,
 you can set the type to "auto", which will create a new type.
 
+Many to many with `through` models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Using `graphene-django-cud`, a `through` model is more akin to a many to one relationship, as it is
+an intermediate model that connects two other models with itself.
+
+If you have a many to many relationship with a `through` model, you can use
+a `many_to_one_extras` field to specify how to handle the `through` model.
+There's also usually no need to specify the `many_to_many_extras` field, as
+the `through` model will be used instead.
```

Comment:
Maybe expand on this to note that the m2m-field will be autogenerated with the input schema, but that it is less useful when using a through-model—and that excluding it might be a good idea.

We should also probably consider auto-excluding the m2m-variants from the schema when encountering a m2m-relation with a `through`.